src/xlsx/extraction/chart_extractor.ts

```diff
@@ -135,29 +141,69 @@ export class XlsxChartExtractor extends XlsxBaseExtractor {
                 label = { text };
               }
             }
-            const color = this.extractChildAttr(
-              chartDataElement,
-              "c:spPr a:solidFill a:srgbClr",
-              "val"
-            );
+            colorElements = this.querySelectorAll(chartDataElement, "c:spPr a:solidFill");
+            const datasetColorElement = colorElements.length
+              ? this.querySelector(colorElements[0], "a:srgbClr")
+              : undefined;
+            const color = datasetColorElement ? datasetColorElement.getAttribute("val") : undefined;
             return {
               label,
               range: this.extractChildTextContent(chartDataElement, "c:val c:f", {
                 required: true,
               })!,
-              backgroundColor: color ? `${toHex(color.asString())}` : undefined,
+              backgroundColor: color ? `${toHex(color.toString())}` : undefined,
+              trend: this.extractChartTrendline(chartDataElement),
             };
           }
         );
       })
       .flat();
   }
 
+  private extractChartTrendline(
+    chartDataElement: Element
+  ): ExcelChartTrendConfiguration | undefined {
+    const trendlineElement = this.querySelector(chartDataElement, "c:trendline");
+    if (!trendlineElement) {
+      return undefined;
+    }
+    const trendlineType = this.extractChildAttr(trendlineElement, "c:trendlineType", "val");
+    const trendlineColor = this.extractChildAttr(trendlineElement, "a:solidFill a:srgbClr", "val");
+    let trendlineConfig: ExcelChartTrendConfiguration = {
+      type: trendlineType ? (trendlineType.asString() as ExcelTrendlineType) : undefined,
+      color: trendlineColor ? `${toHex(trendlineColor.asString())}` : undefined,
+    };
```

Comment:
```suggestion
    const trendlineConfig: ExcelChartTrendConfiguration = {
      type: this.extractChildAttr(trendlineElement, "c:trendlineType", "val")?.asString(),
      ...,
      order: this.extractChildAttr(trendlineElement, "c:order", "val")?.asNum(),
      window: ....
    };
```

do we really need all the if cases ? 

Or we could also simplfy the if cases with 

`trendlineConfig.window = this.extractChildAttr(trendlineElement, "c:period", "val")?.asNum()`